tests/testthat/test_SpatialImage.R

```diff
@@ -42,20 +42,18 @@ test_that("SpatialImage in-memory caching works as expected", {
     img2 <- img[1:500,1:500]
     cache$cached <- c(cache$cached, list(A=img2, B=img2, C=img2))
     dummy <- imgRaster(fspi)
-    if (.Platform$OS.type == "unix") {
-        patha <- paste0("file://", path)
-    } else {
-        patha <- path
-    }
-    expect_identical(names(cache$cached), c(LETTERS[1:3], patha))
+    expect_identical(normalizePath(names(cache$cached), mustWork=FALSE), 
+                    normalizePath(c(LETTERS[1:3], paste0("file://", path)), 
+                                    mustWork=FALSE))
 
     # LRU eviction policy works as intended.
     SpatialExperiment:::.flush_cache()
     oldo <- options("SpatialExperiment.cache.size"=as.double(object.size(img)))
 
     cache$cached <- list(A=img2, B=img2, C=img2)
     img <- imgRaster(fspi)
-    expect_identical(names(cache$cached), paste0("file://", path))
+    expect_identical(normalizePath(names(cache$cached), mustWork=FALSE), 
+                     normalizePath(paste0("file://", path), mustWork=FALSE))
```

Comment:
@HelenaLC are you referring just to this space? or somewhere else too?

Reply:
Yes, this one and the other (I see only 2 changes). The other looks like this on my screen:

```
    expect_identical(normalizePath(names(cache$cached), mustWork=FALSE), 
                    normalizePath(c(LETTERS[1:3], paste0("file://", path)), 
                                    mustWork=FALSE))
```

Reply:
yes, but the other is a multiple of 4. I checked on my rstudio that has vertical lines each 4 spaces

Reply:
Hm, I guess this is not so clear... the guidelines only say "Use 4 spaces for indenting. No tabs." I guess my understanding of this would be something like:
```
expect_identical(
    normalizePath(names(cache$cached), mustWork=FALSE), 
    normalizePath(c(LETTERS[1:3], paste0("file://", path)), mustWork=FALSE))
```
The reason I personally prefer this style is because if you indent within each function multiple times, you will hit 80 character really fast, whereas starting at 4 spaces leaves you enough space & makes it more readable I think.

Reply:
ok, solved